packages/phasor/src/utils/point.ts

```diff
@@ -0,0 +1,90 @@
+import { clamp } from './math-utils.js';
+
+export interface IPoint {
+  x: number;
+  y: number;
+}
+
+export class Point {
```

Comment:
Suggest not using this class. Also refactoring the `edgeless-toolbar` and the newly added `Bound` methods without this dependency sounds better. Otherwise we have too much to migrate for enforcing this convention.

Reply:
ok, looks like the codes are more like functional-style. not OOP style.